common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -999,7 +1000,7 @@ func (d *cassandraPersistence) CreateWorkflowExecution(
 							Msg:              msg,
 							StartRequestID:   protoState.CreateRequestId,
 							RunID:            primitives.UUIDString(protoState.RunId),
-							State:            int(protoState.State),
+							State:            checksumproto.WorkflowExecutionState(protoState.State),
```

Comment:
Do we need to cast here?

Reply:
not anymore! https://github.com/temporalio/temporal/pull/352/files#diff-562b5aad555984585acd7e3b34add537R31 

updated.